mbed-hal/spi_api.h

```diff
@@ -112,54 +166,45 @@ void spi_frequency(spi_t *obj, int hz);
  *
  * @param[in] obj   The SPI peripheral to use for sending
  * @param[in] value The value to send
- * @return Returns the value received during send
+ * @return returns the value received during send
  */
 int  spi_master_write(spi_t *obj, int value);
```

Comment:
Now that I think about it, this PR is about return codes.  Let's keep PRs contained, rather than grouping dissimilar changes together just because a PR is happening.  Removal of blocking APIs should be a single PR.